examples/index.md

```diff
@@ -263,25 +228,17 @@ Content-Type: application/vnd.api+json
     },
     {
       "status": "500",
+      "source": { "pointer": "data/attributes/reputation" },
       "title": "The backend responded with an error",
       "detail": "Reputation service not responding after three requests."
-    },
-    {
-      "status": "400",
-      "detail": "JSON parse error - Expecting property name enclosed in double quotes: line 1 column 2 (char 1)"
```

Comment:
I removed this error here because there's no way a parse error can co-occur with these other errors in the same response. However, I added this example back under the "Advanced source usage" section.